### ISSUE TITLE
Ubuntu 19.04 breaks synergy because of libssl version mismatch

### ISSUE THREAD

**Issue by sorinello** [symless/synergy-core#6483]
### Workaround ###

Installing the Debian package instead of the ubuntu one appears to work

### Operating Systems ###

Server: Xubuntu 19.04 64bit

### Synergy Version ###
synergy_1.10.1.stable_b81+8941241e_ubuntu_amd64.deb

### Steps to reproduce bug ###
1. upgrade your Xubuntu/Ubuntu version from 18.10 to 19.04
2. the synergy package will throw an error during the dist upgrade
3. the synergy package is being proposed to be removed by the installed at the end of the upgrade
4. manually try to install the dpkg package: "sudo dpkg -i 
synergy_1.10.1.stable_b81+8941241e_ubuntu_amd64.deb" will fail with the following error:
```
sorinello@xxx:~$ sudo dpkg -i synergy_1.10.1.stable_b81+8941241e_ubuntu_amd64.deb 
[sudo] password for sorinello: 
Selecting previously unselected package synergy.
(Reading database ... 266412 files and directories currently installed.)
Preparing to unpack synergy_1.10.1.stable_b81+8941241e_ubuntu_amd64.deb ...
Unpacking synergy (1.10.1.stable~b81+8941241e) ...
dpkg: dependency problems prevent configuration of synergy:
 synergy depends on libssl1.0.0 (>= 1.0.1); however:
  Package libssl1.0.0 is not installed.

dpkg: error processing package synergy (--install):
 dependency problems - leaving unconfigured
Processing triggers for hicolor-icon-theme (0.17-2) ...
Processing triggers for gnome-menus (3.32.0-1ubuntu1) ...
Processing triggers for desktop-file-utils (0.23-4ubuntu1) ...
Processing triggers for mime-support (3.60ubuntu1) ...
Errors were encountered while processing:
 synergy
```

### Other info ###

* Problem appeared after migrating to Ubuntu 19.04

**Comment by sorinello** [comment]
After further investigating, I have found a temporary workaround:

1) Do **NOT** run sudo apt-get autoremove. This will remove the dpkg of libssl1.0.0 you had installed before upgrading.
2) check /var/cache/apt/archives/ for a dpkg called **libssl1.0.0_1.0.2n-1ubuntu6.2_amd64.deb**
2) a) if you don't have it download it from this post
[libssl1.0.0_1.0.2n-1ubuntu6.2_amd64.deb.zip](https://github.com/symless/synergy-core/files/3098768/libssl1.0.0_1.0.2n-1ubuntu6.2_amd64.deb.zip)


4) also, you need another package: sudo apt-get install libavahi-compat-libdnssd1
5) install synergy. It should install fine now, without any errors.

Please note that this is a temporary solution until a new version of synergy will be released that will use the newer version of libssl.

**Comment by jimmsta** [comment]
I get an additional problem on a fresh 19.04 install on top of a broken 18.04 install - 
dpkg: error processing archive /home/<username>/Downloads/synergy_2.0.12.beta_b1677+0b61673b_amd64.deb (--install):
 unable to open '/lib/systemd/system/synergy.service.dpkg-new': No such file or directory

touching this 'file' results in the same error message. I did install libssl1.0.0 as per above, and libavahi-compat-libdnssd1. No go. Attempting to compile from source just gives me errors in the source.

**Comment by Jnewbon** [comment]
I have managed to repeat the problem here for v1.10.1_stable, for a quick workaround until then installing the Debian package should work.

**Comment by Jnewbon** [comment]
This is now fixed with the release of v1.10.2-stable.
